crates/aptos/src/common/types.rs

```diff
@@ -1216,11 +1225,12 @@ impl MovePackageDir {
             override_std: None,
             skip_fetch_latest_git_deps: true,
             bytecode_version: None,
-            compiler_version: None,
-            language_version: None,
+            compiler_version: Some(CompilerVersion::latest_stable()),
+            language_version: Some(LanguageVersion::latest_stable()),
             skip_attribute_checks: false,
             check_test_code: false,
-            move_2: false,
+            move_2: true,
+            move_1: false,
```

Comment:
Should there be a move_version field that takes an enum instead? Because I assume it's invalid to set both to true or false?

Reply:
Agree, but perhaps we should have either just keep this as `move_2` with true the default, or remove one of the flags. Enum seems overkill.

Reply:
Hi @banool, I was thinking of the same idea but we may still need to keep the `move-2` flag in case some eco system projects depend on this flag to compile their smart contracts in CI? Enum seems be an overkill because `move-1` will only be used in urgent time and will be removed when V1 retires. Also, external users cannot set both of them to the same value because they are put in the same ArgGroup.